stdlib/public/Concurrency/Executor.swift

```diff
@@ -92,3 +92,27 @@ func _checkExpectedExecutor(_filenameStart: Builtin.RawPointer,
   _reportUnexpectedExecutor(
     _filenameStart, _filenameLength, _filenameIsASCII, _line, _executor)
 }
+
+@available(SwiftStdlib 5.5, *)
+@_silgen_name("swift_task_enqueueOnDispatchQueue")
+internal func _enqueueOnDispatchQueue(_ job: UnownedJob, queue: AnyObject)
```

Comment:
I'm surprised this doesn't need to be `@usableFromInline`

Reply:
The inlinable attributes are actually meaningless because it's all internal